images/win/scripts/Installers/Install-YAMLLint.ps1

```diff
@@ -0,0 +1,3 @@
+pip install yamllint

```

Comment:
I recently found pipx: https://github.com/pipxproject/pipx
That is the tool that allows to install pip packages globally and don't depend on any Python.
I am okay with current way. JFYI